workshops/forms.py

```diff
@@ -630,6 +644,12 @@ class SWCEventRequestForm(forms.ModelForm):
                   '-to-admin-fee.html" target="_blank">Look up administration '
                   'fees</a>.',
     )
+    language = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.LanguageLookup,
+        label='Language',
+        required=False,
+        widget=selectable.AutoComboboxSelectWidget,
```

Comment:
@narayanaditya95 sorry, the way GH displayed the diff it looked as if `PersonCreateForm` was implementing `language` field.

I think for now let's leave EventRequests monolingual, if someone wants to run workshop in other languages they can always add it to the notes.

Unless you implement `Event.languages`, well, then `EventRequest` should have `languages` too.